posts/contributing-to-govuk-design-system.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Why I contribute to the GOV.UK Design System (and why you should too)
+date: 2022-05-03
+tags: post
+layout: post
+author:
+  name: Frankie Roberto
+  url: https://github.com/frankieroberto
+---
+
+Last week a new pattern was published in the GOV.UK Design System that helps users to [confirm a phone number](https://design-system.service.gov.uk/patterns/confirm-a-phone-number/) using a security code sent by text message.
+
+I initiated this contribution almost two years ago when working on a service that was switching to using its own sign in system to avoid relying upon an off-the-shelf authentication service which was hard to maintain.
+
+As the service was quite sensitive, we decided to require a text message confirmation code when signing in – sometimes known as 'two factor authentication'.
+
+The pattern of sending and requesting an SMS code is quite well established, used by many services in and out of government.
+
+However we still had a bunch of detailed design questions. Things like:
+
+* should we call it a ‘security code’, a ‘verification code’, or something else?
+* how long should the codes be valid for?
+* if a code doesn’t arrive straight away and the user requests a new one, should we resend the same code or generate a new one?
```

Comment:
Truly nitpicky details, but I’ve been sticking to dump quotes and letting Smartypants to the hard work (on this website, anyway): 

```suggestion
* if a code doesn't arrive straight away and the user requests a new one, should we resend the same code or generate a new one?
```